src/main/java/ee/ut/logic/Editor.java

```diff
@@ -26,4 +26,20 @@ public static void saveEvent(
         }
         event.setLabel(label);
     }
+
+    public static void saveSettings(
+            Data data,
+            String labelType,
+            String eventSpace,
+            Boolean eventsPacked
+    ){
+        data.setEventSpace(Integer.valueOf(eventSpace));
+        data.setLabelType(labelType);
+        if(eventsPacked){
```

Comment:
It would be safer to also set the value of "packed" to false for all events if the saved settings says so, since otherwise the events would start packed no matter what the final value of this setting is if it has been set to true even once.

Reply:
Additionally, please consider adding a field somewhere, perhaps in this class, that says whether all events should be packed or not and modifying either the Editor logic class (save function) or the Event class default constructor to take this into account.

Reply:
Added variable to Editor class, modified saveEvent function to take it into account